source/reference/minio-mc/mc-tag-set.rst

```diff
@@ -86,6 +86,13 @@ Parameters
 
       mc tag set myminio/mybucket/object.txt "key1=value1&key2=value2"
 
+.. mc-cmd:: --recursive, r
+   :optional:
+
+   .. versionadded:: RELEASE.2023-05-04T18-10-16Z
+
+   Recursively applies the tag to all objects at the specified :mc:`ALIAS <mc tag set ALIAS>`.
```

Comment:
Would you want to apply tags to all the objects at an alias?
Seems like this would only apply usually happen for objects at a bucket or prefix?

Reply:
Made a tweak.